packages/playwright-core/src/server/page.ts

```diff
@@ -744,13 +741,18 @@ export class Page extends SdkObject {
     }
   }
 
-  async setFileChooserIntercepted(enabled: boolean): Promise<void> {
-    this._interceptFileChooser = enabled;
-    await this.delegate.updateFileChooserInterception();
+  async setFileChooserInterceptedBy(enabled: boolean, by: any): Promise<void> {
```

Comment:
Maybe unknown since we only care about identity?
```suggestion
  async setFileChooserInterceptedBy(enabled: boolean, by: unknown): Promise<void> {
```